doc: update the following VM topics (#394)
harvester/docs: docs/vm/clone-vm.md

## Clone VM without volume data

> This operation more like to create a new VM with the same configuration as the source VM.

Author: updated! thanks!!

> This operation is more like creating a new VM with the same configuration as the source VM.

Reviewer: ```suggestion
Cloning a VM without volume data creates a new VM with the same configuration as the source VM.
```
